pkg/cmd/dictionary/entries/clear/clear.go

```diff
@@ -105,20 +123,37 @@ func NewClearCmd(f *cmdutil.Factory, runF func(*ClearOptions) error) *cobra.Comm
 
 // runClearCmd executes the clear command
 func runClearCmd(opts *ClearOptions) error {
+	cs := opts.IO.ColorScheme()
 	client, err := opts.SearchClient()
 	if err != nil {
 		return err
 	}
 
-	dictionnaries := opts.Dictionnaries
-	dictionnariesNames := make([]string, len(dictionnaries))
-	for i, dictionnary := range dictionnaries {
-		dictionnariesNames[i] = string(dictionnary)
+	dictionaries := opts.Dictionnaries
+	dictionariesNames := make([]string, len(dictionaries))
+	dictionariesCustomEntriesNb := make([]int, len(dictionaries))
+	for i, dictionnary := range dictionaries {
```

Comment:
Just a question there: what's the rule of thumb to know where to write "validation" logic? `runE` should validate and prepare all data so `run` can assume everything has been checked?

Reply:
IMHO, we should use `runE`to run every validation logic for the flags / the arguments and return on errors.
Here, I'm not returning an error but a warning that the provided dictionaries do not contains any entries (the command ran successfully, but didn't had any effect)